i18n/en.json

```diff
@@ -5,5 +5,8 @@
 			"Morne Alberts"
 		]
 	},
-	"pageapprovals-description": "Allows approvers to make pages as approved"
+	"pageapprovals-description": "Allows approvers to make pages as approved",
+	"pageapprovals-status-approved": "This page is approved.",
```

Comment:
Check task/requirements: we need more info in here: user and time

Reply:
I don't know if you omitted on purpose. We don't need to get everything in the first PR. I'm just making sure you are aware

Reply:
Yes. I am aware of this. There are still a lot of missing pieces that we need to put together and combine logic from other areas of the code. Those English translations for now are just a placeholder, they will likely get more complex in our coming PRs.

Reply:
typo: "...to mark pages..."

Reply:
Blind devs are blind :D

Reply:
I'll correct the **typo**, but it was already, its harder to spot since I primarily focus on reviewing the changes introduced in the current PR.